features/index/format_as_csv.feature

```diff
@@ -79,6 +79,38 @@ Feature: Format as CSV
       | title | body |
       | Hello, World | (.*) |
 
+  Scenario: With humanize_name option turned off globally
+    Given a configuration of:
+    """
+      ActiveAdmin.application.csv_options = { humanize_name: false }
+      ActiveAdmin.register Post do
+      end
+    """
+    And a post exists
+    When I am on the index page for posts
+    And I follow "CSV"
+    And I should download a CSV file with "," separator for "posts" containing:
```

Comment:
s/And/Then/

Reply:
Changed last "And" to "Then" now it reads like a sentence if you ignore the configuration block.